EDDiscovery/Trilateration/TrilaterationControl.cs

```diff
@@ -731,12 +731,12 @@ private void toolStripButtonMap_Click(object sender, EventArgs e)
         {
             var centerSystem = TargetSystem;
             if (centerSystem == null || !centerSystem.HasCoordinate) centerSystem = LastKnownSystem;
-            FormMap map2 = new FormMap(centerSystem, _discoveryForm.SystemNames)
-            {
-                ReferenceSystems = CurrentReferenceSystems.ToList(),
-                visitedSystems = _discoveryForm.visitedSystems
-            };
-            map2.Show();
+            var map = _discoveryForm.Map;
+
+            map.Instance.Reset();
+            map.Instance.CenterSystem = centerSystem;
+            map.Instance.ReferenceSystems = CurrentReferenceSystems.ToList();
+            map.Show();
```

Comment:
See how the map is interacted with now? First we set a few properties in the FormMap direct, then call the "show" method in the proxy class (`MapManager`)to manage the process of recalling the map.